pkg/pillar/containerd/containerd.go

```diff
@@ -76,6 +82,9 @@ func InitContainerdClient() error {
 	if ctrdCtx == nil {
 		ctrdCtx = namespaces.WithNamespace(context.Background(), ctrdServicesNamespace)
 	}
+	if ctrdSystemCtx == nil {
+		ctrdSystemCtx = namespaces.WithNamespace(context.Background(), ctrdSystemServicesNamespace)
+	}
```

Comment:
You mean doing it as a transaction @adarsh-zededa ? So that we ever have both of them initialized or not? (but not a situation where one is initialized and the other one isnt).

Reply:
Yes @rvs , `verifyCtr()` was added to check if Containerd was initialized properly before calling Containerd APIs.